.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,45 @@
+name: deploy
+
+on:
+  workflow_run:
+    workflows:
+      - check
+    types:
+      - completed
+    branches:
+      - dev
+
+# Ensures that only one deploy task per branch/environment will run at a time.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  cdk-deploy:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    permissions:
+      id-token: write
+      contents: read
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Assume AWS Role
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: us-east-1
+          role-to-assume: TBD
```

Comment:
The role is missing

Reply:
yes that is correct.  The role will be generated by the dependent PR.  Could you please have a look at that one as well?

Reply:
I did, see 👀 emoji added to the OP. I'm not familiar with the content of the PR but if you want I can still approve the PR.

Reply:
thanks for taking a look.  do whatever you are comfortable with.